src/localstore/incremental/JasmineGraphIncrementalLocalStore.cpp

```diff
@@ -11,31 +11,70 @@ See the License for the specific language governing permissions and
 limitations under the License.
  */
 
-#include <memory>
-#include <stdexcept>
 #include "JasmineGraphIncrementalLocalStore.h"
+
+#include <memory>
 #include <nlohmann/json.hpp>
+#include <stdexcept>
 using json = nlohmann::json;
 
 #include "../../util/logger/Logger.h"
+#include "../../centralstore/incremental/RelationBlock.h"
+
 
 Logger incremental_localstore_logger;
 
-JasmineGraphIncrementalLocalStore::JasmineGraphIncrementalLocalStore(unsigned int graphID, unsigned int partitionID){
+JasmineGraphIncrementalLocalStore::JasmineGraphIncrementalLocalStore(unsigned int graphID, unsigned int partitionID) {
     gc.graphID = graphID;
     gc.partitionID = partitionID;
-    gc.maxLabelSize = 10;
-    gc.openMode = "trunk";
+    gc.maxLabelSize = 43; // TODO tmkasun: read from .properties file
+    gc.openMode = "trunk"; // TODO tmkasun: read from .properties file
     this->nm = new NodeManager(gc);
 };
 
-std::string JasmineGraphIncrementalLocalStore::addEdgeFromString(std::string edgeString)
-{
+std::string JasmineGraphIncrementalLocalStore::addEdgeFromString(std::string edgeString) {
     try {
-        auto edge = json::parse(edgeString);
-        NodeBlock *sourceNode = this->nm->addNode(edge["id"]);
-    } catch (const std::exception &) {
+        auto edgeJson = json::parse(edgeString);
+
+        auto sourceJson = edgeJson["source"];
+        auto destinationJson = edgeJson["destination"];
+
+        std::string sId = std::string(sourceJson["id"]);
```

Comment:
Lets change `sId` to `sourceId` and `dId` to `destinationId`

Reply:
Renamed in this commit https://github.com/miyurud/jasminegraph/pull/132/commits/d8d887f5453408266df16065d675be6691184b72

Reply:
Ok